pkg/operator/starter.go

```diff
@@ -111,13 +106,31 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 		return err
 	}
 
+	secretSyncController := secret.NewSecretSyncController(
+		operatorClient,
+		kubeClient,
+		kubeInformersForNamespaces,
+		util.Resync,
+		controllerConfig.EventRecorder)
+	vpcBlockCSIController := csidriver.NewVPCBlockController(
```

Comment:
I am trying to understand what `NewVPCBlockController` does. It seems to me it just blindly starts `secretSyncController` and `csiControllerSet` on the every `sync()`. 

This *seems* to be unnecessary. Can you call `go csiControllerSet.Run()` and `go secretSyncController.Run()` just fine below. And delete whole `VPCBlockController`?

Reply:
removed